diff --git a/src/settings/mangareaderui.rc b/src/settings/mangareaderui.rc
index 3840d0a..72730d8 100644
--- a/src/settings/mangareaderui.rc
+++ b/src/settings/mangareaderui.rc
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <gui name="mangareader"
-     version="1"
+     version="2"
      xmlns="http://www.kde.org/standards/kxmlgui/1.0"
      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
      xsi:schemaLocation="http://www.kde.org/standards/kxmlgui/1.0
