diff --git a/src/main.cpp b/src/main.cpp
index 5c55f9a..0959462 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -34,7 +34,7 @@ int main(int argc, char *argv[])
     KAboutData aboutData;
     aboutData.setDisplayName(i18n("Manga Reader"));
     aboutData.setComponentName(QStringLiteral("mangareader"));
-    aboutData.setVersion(QStringLiteral("1.1.0").toUtf8());
+    aboutData.setVersion(QStringLiteral("1.2.0").toUtf8());
     aboutData.setShortDescription(i18n("Manga reader for local files."));
     aboutData.setLicense(KAboutLicense::GPL_V3);
     aboutData.setCopyrightStatement(i18n("(c) 2019-2020"));
diff --git a/src/view.cpp b/src/view.cpp
index 615ff77..6753e15 100644
--- a/src/view.cpp
+++ b/src/view.cpp
@@ -399,7 +399,7 @@ void View::contextMenuEvent(QContextMenuEvent *event)
     if (QGraphicsItem *item = itemAt(position)) {
         page = qgraphicsitem_cast<Page *>(item);
         auto menu = new QMenu();
-        menu->addSection(i18n("Page %1").arg(page->number() + 1));
+        menu->addSection(i18n("Page %1", page->number() + 1));
 
         QString zoomActionText = page->zoom() == 1.0
                 ? i18n("Zoom In")
